docs/monitoring/inventory.md

```diff
@@ -1,13 +1,105 @@
+- Number of different OS distributions,
+- Number of different OS versions,
+- A number of different kernel versions.
+
+![Inventory Info](../images/monitoring/inventory_inventory_info.png)
+
+### Inventory Overview
+
+The Inventory overview provides a per-host view of each host, virtual machine, and container along with the operating system and its version, number of CPUs, their type, memory, number of hard disk drives and total capacity.
+
+![Inventory Overview](../images/monitoring/inventory_overview.png)
+
+Expanding each row gives detailed information about the system including:
+
+- Unique identifier of the system,
+- OS version,
+- Kernel version,
+- Java virtual machine version,
+- System type (bare metal, virtual machine, container),
+- Total memory size, 
+- Memory slot information,
+- Total disk size, 
+- Hard disk drives information.
+
+In addition, an overview of all the installed packages on a given system is available. This includes:
+
+- Installation source, 
+- Package name, 
+- Package version.
+
+![Inventory Overview](../images/monitoring/inventory_packages.png)
+
+### Comparing Hosts
+
+The Inventory UI allows comparing different systems to quickly and efficiently see differences between them:
+
+![Inventory Package Comparison](../images/monitoring/inventory_package_comparison.png)
+
+## How does it Work?
+
+The [Sematext Agent](../agents/sematext-agent) keeps track of all changes done on the file system and displaying them in the Inventory Monitoring UI when installing, upgrading or removing packages on the host system, in a virtual machine or container with:
+
+- Node.JS (when using NPM package manager)
+- Python (when using PIP package manager)
+- DEB (for example, when using APT package manager)
+
+The Agent will periodically check the state of the packages and ship the information to Sematext Cloud ensuring the state of the packages is always up to date and visible.
+
+## Enabling Inventory Monitoring
+
+To enable inventory monitoring you need to adjust your `st-agent.yml` file located in the `/opt/spm/properties` directory and set the following section:
+
+```
+pkg:
+  enabled: true
+  interval: 10m
+```
+
+After that the [Sematext Agent](../agents/sematext-agent) needs to be restarted by running the following command:
+
+```
+service spm-monitor-st-agent restart
+```
+
+## Gathered Data
+
+The [Sematext Agent](../agents/sematext-agent) gathers the following data about the system:
+
+- OS version,
+- Kernel version,
+- Java virtual machine version,
+- System type (bare metal, virtual machine, container),
+- Total memory size, 
+- Memory slot information,
+- Total disk size, 
+- Hard disk drives information.
+
+In addition, the following information about the packages is gathered for NodeJS, Python and DEB sources:
+
+- Installation source, 
+- Package name, 
+- Package version.
+
+## Solving Problems With Inventory Monitoring
+
+Inventory monitoring allows quickly solving the following issues:
```

Comment:
Inventory monitoring allows quickly solving the following issues:
=> Here are some of the common use cases and issues that Inventory monitoring helps solve:

Reply:
Adjusted.